docs-wiki/Build-and-test.md

```diff
@@ -0,0 +1,224 @@
+
+To learn more, see [Workbook modules](https://github.com/microsoft/finops-toolkit/blob/dev/src/workbooks/README.md).
+
+### Building Bicep Registry modules
+
+Bicep Registry modules support a custom "language" that enables reusing code across target scopes. Bicep Registry modules designed to support multiple scopes will generate multiple output folders for each module.
+
+To build all bicep modules, run:
+
+```powershell
+cd "<repo-root>"
+src/scripts/Build-Toolkit
+```
+
+To build only a single module, run:
+
+```powershell
+cd "<repo-root>"
+src/scripts/Build-Bicep ..\bicep-registry\<module>
+```
+
+Bicep Registry modules include a `main.test.bicep` file that includes all tests. To test Bicep Registry modules, run:
+
+```powershell
+cd "<repo-root>"
+src/scripts/Deploy-Toolkit "<module-name>" -Build -Test
+```
+
+To learn more, see [Bicep Registry modules](https://github.com/microsoft/finops-toolkit/blob/dev/src/bicep-registry/README.md).
+
+### Building open data files
+
+The build script for open data files generates the resource types file from Azure portal metadata and then generates PowerShell commands and tests for each file.
+
+To build open data, run:
+
+```powershell
+src/Build-OpenData
+```
+
+To learn more about open data, see [Open data](https://github.com/microsoft/finops-toolkit/blob/dev/src/open-data/README.md).
+
+### Building PowerShell
+
+The PowerShell module generates a single module file from functions created in separate files.
+
+To build the PowerShell module, run the following as an administrator:
+
+```powershell
+cd "<repo-root>"
+src/scripts/Build-PowerShell
+```
+
+To install the PowerShell module locally, run:
+
+```powershell
+cd "<repo-root>"
+Remove-Module FinOpsToolkit -ErrorAction SilentlyContinue
+Import-Module -FullyQualifiedName src/powershell/FinOpsToolkit.psm1
+```
+
+The Build-PowerShell script calls an Invoke-Build task internally.
+
+> [!NOTE]
+> We have a partial implementation of Invoke-Build but this has not been fully  implemented, so there is a mix of custom scripts and Invoke-Build scripts. If anyone has experience in this area, we would love to get help centralizing on a single build system.
+
+### Building Power BI reports
+
+Power BI reports cannot be built automatically. Generating PBIT and PBIX files must be done manually:
+
+1. Open the desired Power BI report.
+2. Save the report as a `.pbix` file using the same name in the `releases` folder.
+3. Remove unnecessary queries.
+4. Remove setup instructions for unnecessary parameters.
+5. Remove unnecessary parameters.
+6. Save the report again.
+7. Copy the first paragraph description from the main page.
+8. Save the report as a `.pbit` file using the copied description and add "To learn more, see https://aka.ms/ftk/<report-name-no-spaces>".
+
+### Building documentation
```

Comment:
@sebassem Added this based on your request. It was a long time coming, so thanks for the nudge 😜